js/src/personal/amanda/Amanda.module.css

```diff
@@ -1,3 +1,12 @@
+.landingPage {
+    width: 100vw;
+    height: 100vh;
+
+}
+.mainPage {
+    display: flex;
+    margin-top: 50px;
```

Comment:
I think rem(50px) might be best practice here, but I'm not fully sure about the limitations and benefits of rems.
https://mantine.dev/styles/rem/

Maybe Celina may know more about them, but likely good to go with the rem(px)